lldb/test/API/functionalities/data-formatter/data-formatter-stl/generic/multimap/TestDataFormatterGenericMultiMap.py

```diff
@@ -231,23 +244,39 @@ def cleanup():
             "frame variable is",
             substrs=[
                 multimap,
-                "size=4",
-                '[0] = (first = 1, second = "is")',
-                '[1] = (first = 2, second = "smart")',
-                '[2] = (first = 3, second = "!!!")',
-                '[3] = (first = 85, second = "goofy")',
+                'size=4',
```

Comment:
What is it about this change that is defeating the ValueObject printer from compressing this output onto one line?  It looks like the contents that get printed are the same, so there's something about switching from a Summary provider to a child provider that's causing problems.  We should fix that as people are really picky about variable printing being as space efficient as possible.